docs/mobile-apps/automated-testing/appium/appium-2-deque-drivers.md

```diff
@@ -0,0 +1,183 @@
+ensuring your apps are inclusive for all users.
+
+With this integration, you can:
+
++ Detect and address accessibility issues early in the development cycle.
++ Automate mobile accessibility testing using Sauce Labs' secure and scalable cloud infrastructure.
++ Gain actionable insights to improve compliance with global accessibility standards.
+
+The following guide explains how to set up Sauce Labs’ integration with [axe DevTools® for Mobile](https://docs.deque.com/devtools-mobile/appium) 
+and leverage its powerful tools to optimize accessibility testing in your mobile apps.
+
+
+
+
+## What You'll Need
+
+- A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
+- Your Sauce Labs [Username and Access Key](https://app.saucelabs.com/user-settings)
+- Familiarity writing and running [Appium tests](/mobile-apps/automated-testing/appium/)
+- [A Deque Account](https://axe.deque.com/plans) (axe DevTools® for Mobile Product is required) 
+
+
+## Appium 2 Version: `appium2-deque-accessibility`
+
+Sauce Labs offers various versions of Appium 2 in our cloud. One of these is the `appium2-deque-accessibility` version, 
+designed to support accessibility testing on mobile apps for both iOS and Android. This version integrates Appium drivers developed by 
+the Deque team, enabling seamless integration of accessibility scans into your existing test automation workflows.
+
+You can find more details on Sauce Labs' [Appium versions documentation](/mobile-apps/automated-testing/appium/appium-versions/#appium-2x).
+
+## Key Features of `appium2-deque-accessibility`
+
+- ****Includes Deque Drivers :**** This version incorporates the following accessibility drivers provided by Deque:
+    - axe-appium-uiautomator2-driver: Automation name is `AxeUIAutomator2`
+    - axe-appium-xcuitest-driver: Automation name is `AxeXCUITEST`
+:::info Min iOS Version for `axe-appium-xcuitest-driver`:
+- You can use the `axe-appium-xcuitest-driver` only for devices with iOS 17 or above. 
+- To ensure compatibility, set the `appium:platformVersion` capability to `'^1(7|8).*$'` in your Appium configuration.
+:::
+- ****Backward Compatibility with Deprecated Plugin:**** While Deque’s  [axeDevToolsMobile Appium Plugin](https://docs.deque.com/devtools-mobile/2024.2.14/en/june-2024-3) 
+has been deprecated, Sauce Labs will continue hosting it until January 31st 2025, allowing users time to migrate to the new drivers.
+
+## How to Use `appium2-deque-accessibility`
+
+Follow these steps to set up and execute accessibility tests using the `appium2-deque-accessibility` image, leveraging Deque's Appium drivers for Android and iOS.
+
+1. Set Up Your Appium Capabilities
+
+<Tabs
+groupId="appium2-deque-accessibility"
+defaultValue="AxeUIAutomator2"
+values={[
+{label: 'AxeUIAutomator2', value: 'AxeUIAutomator2'},
+{label: 'AxeXCUITEST', value: 'AxeXCUITEST'},
+]}>
```

Comment:
I tested these tabs in the preview, they do not work

Reply:
Seems my preview was broken its fixed now

Reply:
Please check again, it works for me. 
https://docs.dev.saucelabs.net/pr-preview/pr-3013/mobile-apps/automated-testing/appium/appium-deque-accessibility-testing/